Atomic/atomic.py

```diff
@@ -331,8 +331,8 @@ def _inspect_image(self, image=None):
         except docker.errors.APIError:
             pass
         except requests.exceptions.ConnectionError as e:
-            raise IOError("Unable to communicate with docker daemon: %s\n" %
-                          str(e))
+            raise IOError("Cannot connect to the Docker daemon. Is the docker daemon running on this host?")
+
```

Comment:
The original message here was good enough. If you want to leave the question in there that is fine but might still be useful to have the `str(e)` in there. 

Either way works. Make the change if you want or leave it like is.